e2e/resolve-conditions/browser-env.js

```diff
@@ -1,16 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-'use strict';
-
-const BrowserEnv = require('jest-environment-jsdom');
-
-module.exports = class BrowserEnvWithConditions extends BrowserEnv {
```

Comment:
don't delete the test, change it to use some other custom condition

Reply:
I've replaced the `browser` condition for a new `deno` one, as defined at https://nodejs.org/api/packages.html#community-conditions-definitions, since there's no `jest-environment-deno` package, so it was perfect to test a "custom" environment.

Reply:
perfect 👍